pkgs/development/libraries/qtspell/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  doxygen,
+  enchant,
+  glib,
+  kdePackages,
+  llvmPackages,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qtspell";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "manisandro";
+    repo = "qtspell";
+    rev = "${version}";
+    hash = "sha256-yaR3eCUbK2KTpvzO2G5sr+NEJ2mDnzJzzzwlU780zqU=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    doxygen
+    pkg-config
+    kdePackages.qttools
+  ];
+
+  buildInputs =
+    [ enchant ]
```

Comment:
qtbase is not missing now, it may  be propagated by qttools as a dependency. We should not rely on this behavior, qttool is nativeBuildInputs, and may not find qtbase when cross-compiling

Should add 'qtbase' here